index.html

```diff
@@ -11,9 +11,18 @@
     <title>KTP Strikes Fall 2019</title>
   </head>
   <body>
-
-    <table id="table">
-      <thead>
+    <div id="title" class="py-3">
+      <h1 class="text-center">Kappa Theta Pi Strikes</h1>
+      <h5 class="text-center">A log of all strikes added and removed and their reasons why.</h5>  
+    </div>
+    <div id="loading-message" class="text-center mt-2">
+      <h5 class="py-2">Loading Table...</h5>
+      <div class="spinner-border text-primary" role="status">
+        <span class="sr-only">Loading...</span>
+      </div>  
+    </div>
+    <table id="table" class="table table-striped" style="display: none;">
```

Comment:
Why is this style inline?

Reply:
Using the bootstrap display none class resulted in the table not appearing upon using jQuery's fadeIn() function, so an inline worked best. Obviously a style tag in the header would work as well but I maximized bootstrap use.